ansible/roles/cumulus/tasks/main.yml

```diff
@@ -58,8 +59,26 @@
     - update_ips
 
 - name: Create config.json
-  action: template src=config.json.j2 dest=/opt/hpccloud/cumulus/config.json mode=640 owner=hpccloud group=celery
+  action: template src=config.json.j2 dest=/opt/hpccloud/cumulus/cumulus/conf/config.json mode=640 owner=hpccloud group=celery
   tags:
     - cumulus
     - update_ips
   sudo: yes
+
+- name: Install cumulus
+  pip:
+    chdir: /opt/hpccloud/cumulus
+    name: .
+    extra_args: "--user"
+  become_user: hpccloud
+  when: development == False
+  tags: cumulus
+
+- name: Install cumulus (Development)
+  pip:
+    chdir: /opt/hpccloud/cumulus
+    name: .
+    extra_args: "-e"
+  sudo: true
+  when: development == True
```

Comment:
would just 'development' work here and 'not development' above. I am always a little wary of comparing things to True ....